tests/e2e/core-tests/orders_02_multiple_test.js

```diff
@@ -1,7 +1,7 @@
 const helpers = require('../pages/helpers');
 Feature('Multiple orders From Single Vendor Functionality');
-Scenario('Customer Order Multiple Product from signle vendor', ({ I }) => {
-    I.loginAsCustomer();
+Scenario('Customer Order Multiple Product from signle vendor', ({ I ,loginAs}) => {
+    loginAs('customer');
```

Comment:
Maintain upper and lower case for the user role because the user is case sensitive.